ompi/mca/bcol/basesmuma/bcol_basesmuma_buf_mgmt.c

```diff
@@ -99,6 +99,9 @@ int bcol_basesmuma_free_buff( sm_buffer_mgmt * buff_block,
      *   associated with this bank have been freed.
      */
     assert(generation == buff_block->ctl_buffs_mgmt[memory_bank].bank_gen_counter);
+#if !OPAL_ENABLE_DEBUG
```

Comment:
heck if I care - I just used a technique that had been used in a number of other similar places in the code